openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -61,6 +61,16 @@ def create_jwt_from_token(token_dict, oauth_adapter, use_asymmetric_key=None):
     access_token = oauth_adapter.get_access_token(token_dict['access_token'])
     client = oauth_adapter.get_client_for_token(access_token)
 
+    # .. setting_name: JWT_ACCESS_TOKEN_EXPIRE_SECONDS
+    # .. setting_default: 60 * 60
+    # .. setting_description: The number of seconds a JWT access token remains valid. We use this
+    #     custom setting for JWT formatted access tokens, rather than the django-oauth-toolkit setting
+    #     ACCESS_TOKEN_EXPIRE_SECONDS, because the JWT is non-revocable and we want it to be shorter
+    #     lived than the legacy Bearer (opaque) access tokens, and thus to have a smaller default.
+    # .. setting_warning: For security purposes, 1 hour (the default) is the maximum recommended setting
+    #     value. For tighter security, you can use a shorter amount of time.
```

Comment:
@timmc-edx: [inform] I didn't want to state anything explicit on the lower end, like 5 minutes, because this would affect IDA to IDA calls as well, and I don't want to test to see if this really is a reasonable setting.